games.zetsubou.WanderNoMore.metainfo.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>games.zetsubou.WanderNoMore</id>
+	<metadata_license>CC-BY-SA-3.0</metadata_license>
```

Comment:
CC0-1.0 is largely prefered for metadata (I don't think this one is accepted either)